packages/ui/src/components/Activity/ActivityListItem.tsx

```diff
@@ -61,25 +61,31 @@ export function ActivityListItemContent({
   const channel: db.Channel | undefined = newestPost.channel ?? undefined;
   const modelUnread =
     summary.type === 'post'
-      ? newestPost.channel?.unread ?? null
+      ? (newestPost.channel?.unread ?? null)
       : summary.type === 'group-ask'
-        ? newestPost.group?.unread ?? null
-        : newestPost.parent?.threadUnread ?? null;
+        ? (newestPost.group?.unread ?? null)
+        : (newestPost.parent?.threadUnread ?? null);
   const { data: unread } = store.useLiveUnread(modelUnread);
   const unreadCount = useMemo(() => {
     return (isGroupUnread(unread) ? unread.notifyCount : unread?.count) ?? 0;
   }, [unread]);
 
-  const title = !channel
-    ? group
-      ? group.title ?? ''
-      : ''
-    : channel.type === 'dm'
-      ? 'Direct message'
-      : channel.type === 'groupDm'
-        ? 'Group chat'
-        : (group?.title ? group.title + ': ' : '') +
-          getChannelTitle(channel, calm.disableNicknames);
+  const channelTitle = useChannelTitle(channel ?? null);
+  const title = useMemo(() => {
+    if (channel == null || channelTitle == null) {
+      return group?.title ?? '';
+    }
+    if (channel.type === 'dm') {
+      return 'Direct message';
+    }
+    if (channel.type === 'groupDm') {
+      return 'Group chat';
+    }
+    if (group?.title) {
+      return `${group.title}: ${channelTitle}`;
+    }
+    return channelTitle;
+  }, [channelTitle, channel, group]);
```

Comment:
I made this change because I wanted to control this with the post collection config. I did so in f76dd969701437d5d54f77e353375b6e95b3cf87 but it felt weird to me – I think I don't get this feature – reverted the commit.